script.skinshortcuts/default.py

```diff
@@ -307,11 +304,22 @@ def _manage_shortcuts( self, group, defaultGroup, nolabels, groupname ):
         if homeWindow.getProperty( "skinshortcuts-loading" ) and int( calendar.timegm( gmtime() ) ) - int( homeWindow.getProperty( "skinshortcuts-loading" ) ) <= 5:
             return
 
+        # Get current Super Favourites context menu status
+        sfMenu = None
+        if xbmc.getCondVisibility( "System.HasAddon(plugin.program.super.favourites)" ):
```

Comment:
I'm not sure if it's OK to change another addon's settings, at least not without first alerting the user and asking for permission from him/her.  Doesn't feel right.